LayoutTests/imported/w3c/web-platform-tests/css/css-shapes/shape-functions/circle-function.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>CSS Shapes Module Level 1: parsing the circle() function</title>
+<link rel="help" href="https://drafts.csswg.org/css-shapes-1/#funcdef-basic-shape-circle">
+<meta name="assert" content="Tests parsing of the circle() function">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/css/support/parsing-testcommon.js"></script>
+<style>
+    #target {
+        width: 200px;
+        height: 100px;
+    }
+</style>
+</head>
+<body>
+<div id="target"></div>
```

Comment:
Can you be consistent on whether you include the target or not?

I'd prefer not including it.